apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/PipelineApplication.java

```diff
@@ -167,6 +168,14 @@ public static void main(String[] args) throws Exception {
 
       final RdaLoadOptions rdaLoadOptions = appConfig.getRdaLoadOptions().get();
 
+      final Optional<RdaServerJob> mockServerJob = rdaLoadOptions.createRdaServerJob();
+      if (mockServerJob.isPresent()) {
+        pipelineManager.registerJob(mockServerJob.get());
+        LOGGER.warn("Registered RdaServerJob.");
```

Comment:
Because if we meant to configure this mode in a particular environment then it's not an error but it is unusual to have it enabled so I thought I'd make it easy to detect that we had done so by making it a warning.